pkg/server/download.go

```diff
@@ -235,9 +235,12 @@ func fileInfoPacked(ctx context.Context, sh *search.Handler, src blob.Fetcher, r
 		log.Printf("ui: fileInfoPacked: skipping fast path due to error from WholeRefFetcher (%T): %v", src, err)
 		return fileInfo{whyNot: "WholeRefFetcher error"}, false
 	}
-	modtime := fi.ModTime
-	if modtime.IsAnyZero() {
-		modtime = fi.Time
+
+	modtime := time.Time{}
```

Comment:
I'd write `var modtime time.Time` which has fewer symbols and looks more like what it is: a variable declaration

Reply:
good point, looks better that way.